lib/tcramer.rb

```diff
@@ -8,5 +8,6 @@ module Tcramer
     'Agile groove',
     '🐹',
     "That's a great question, what do YOU think?"
+    "What's your opinion of walruses?"
```

Comment:
Pls add a line-ending comma on the prior line! Reminds me that we need a test suite for tcramer. :laughing:

Reply:
That would be lovely. I was just starting to wonder about these things.